Stresser/RegistryArtifact.cpp

```diff
@@ -0,0 +1,43 @@
+#include "RegistryArtifact.h"
+
+RegistryArtifact::RegistryArtifact(std::string name, std::string data) : IArtifact(name, data)
+{
+	this->registryPrefix = RegistryArtifactUtils::getRegistryPrefix(this->data);
+	this->registrySubKey = RegistryArtifactUtils::getRegistrySubKey(this->data);
+	this->install();
+}
+
+RegistryArtifact::~RegistryArtifact()
+{
+	this->uninstall();
+}
+
+void RegistryArtifact::install() const
+{
+	HKEY hKey;
+
+	auto createRegStatus = RegCreateKeyExA(
```

Comment:
Change return value type to `LSTATUS `

Use `auto` only when the right side clearly indicates the return type, such:
`auto section = reinterpret_cast<PSECTION>(buffer)`
For better code clarity.